packages/application-components/src/components/internals/tabular-page.tsx

```diff
@@ -58,7 +67,13 @@ const FormControlsContainer = styled.div`
 `;
 
 const CustomViewsSelectorWrapper = styled.div`
+  // Bear in mind the margins are dependant on the context the component
+  // is rendered in. For example, when rendered in a custom view, its panel
+  // already contains default paddings, so we don't need to add margins here.
   margin: ${appKitDesignTokens.marginForCustomViewsSelectorAsTabular};
+  body[data-extension-type='custom-view'] & {
+    margin: ${appKitDesignTokens.marginForCustomViewsSelectorAsTabularInCustomView};
+  }
```

Comment:
I guess this also needs to be adjusted

Reply:
I forgot about that one. Sorry.

Updated here: 129d8b9fd8971c2518527446da20ee929eacaa27